ecommerce-product-page-main/styled.css

```diff
@@ -200,98 +197,65 @@ body{
     background-color: hsl(26,100%,55%);
 }
 
-
-
-img.space{
-    margin-top: 12px;
-    width: 130px;
-}
-
-ul.list{
-
+.list {
     display: flex;
-    list-style: none;
+    text-decoration: none;
+    gap: 10px;
+    align-items: center;
 }
 
 
-
-ul.list li+li:before{
-
-    padding: 10px;
-    content: " ";
-}       
-
-ul.list li a{
-
+.list a{
     color: hsl(219, 9%, 45%);
     text-decoration: none;
-
 }
 
 
-ul.list li a:hover{
-    padding-bottom: 38px;

```

Comment:
Here i used the padding-bottom to create that space between the menu words(men women etc) and the yelow line that shows when mouse is hovered over them.

What did you use to ger this effect please?

Reply:
i used gap.. it is a flex box